File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/VerticalCellViewHolder.java
package org.fossasia.susi.ai.chat.adapters.viewholders;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import org.fossasia.susi.ai.R;

import butterknife.BindView;
import butterknife.ButterKnife;

/*
 * A view item for the individual recycler view element inside the TabViewHolder
 * */

public class VerticalCellViewHolder extends RecyclerView.ViewHolder {

    @BindView(R.id.column)
    public TextView column;
    @BindView(R.id.data_item)
    public TextView data;
    @BindView(R.id.data_item_link)
    public TextView linkData;

    public VerticalCellViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/VerticalCellViewHolder.kt
package org.fossasia.susi.ai.chat.adapters.viewholders

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import kotterknife.bindView
import org.fossasia.susi.ai.R

class VerticalCellViewHolder(itemView: View) : androidx.recyclerview.widget.RecyclerView.ViewHolder(itemView) {

    val column: TextView by bindView(R.id.column)
    val data: TextView by bindView(R.id.data_item)
    val linkData: TextView by bindView(R.id.data_item_link)
}
